### PR TITLE
gds

### PR DESCRIPTION
**Pull request by jaymshah3**
this will be the p/r to manage refactoring code so it uses the GDS model we discussed ove rmsger.

### CODE REVIEW

**Comment by jaymshah3** [comment]
to do:
edit web_driver to utilize the global map + room_id to set and retrieve global variables.

File: web/GameDataService.py [hunk 1]
```diff
@@ -2,15 +2,43 @@
...
+        else:
+            del self.gds_map[room_id]
+
+    def get_game_data(self,room_id):
+        if room_id not in self.gds_map.keys():
+            raise ValueError("room_id does not exist")
+        else:
+            return self.gds_map[room_id]
+
 class GameState(Enum):
```
Comment (adshastri) [inline web/GameDataService.py:33]: this is defined here and in web_driver

File: web/web_driver.py [hunk 2]
```diff
@@ -33,233 +35,205 @@ class GameState(Enum):
 number_of_all_ins = 0
 big_blind_action = False
 aggressors = []
+room_to_gds = GameDataService.room_to_gds
```
Comment (adshastri) [inline web/web_driver.py:38]: you dont use any of these globals now right

File: web/web_driver.py [hunk 3]
```diff
@@ -33,233 +35,205 @@ class GameState(Enum):
...
+
 
+def decorator_action(function):
```
Comment (adshastri) [inline web/web_driver.py:41]: can you change this to `needs_room`

**Pull request merged**